wappuapp-adminpanel/Dockerfile.web

```diff
@@ -0,0 +1,24 @@
+# Builder
+FROM node:8.7.0-alpine as builder
+
+WORKDIR /app
+
+COPY package-lock.json .
+COPY package.json .
+
+RUN npm install
+
+COPY src ./src
+COPY public ./public
+COPY build ./build
+
+RUN npm run build
+
+FROM nginx:1.13.5-alpine
+
+WORKDIR /usr/share/nginx/html
+
+COPY nginx/nginx.heroku.conf /etc/nginx/nginx.conf.template
+COPY --from=builder  /app/build .
+
+CMD /bin/sh -c "envsubst '\$BACKEND_URI,\$PORT,\$FRONTEND_URI' < /etc/nginx/nginx.conf.template > /etc/nginx/nginx.conf && nginx -g 'daemon off;'"
```

Comment:
Github valittaa "No new line at end of file"

Reply:
korjataan